pressbooks/pressbooks
# Store download event data

Issue: #2203 

This PR adds a new feature to record book download events. It will not record any sensitive information.
It creates a generic `pressbooks_tracking` table and adds a `BookDownload.php` class that is responsible for registering the download tracking action.

#### How to test

With the option to download books enabled
* Go to the book home page and download any file format available
* The download should work like before
* Check the database and see if there is a new table `wp_pressbooks_tracking`
* Check if the table has download records
